src/main/java/org/jboss/windup/operator/model/WindupResourceSpec.java

```diff
@@ -0,0 +1,89 @@
+package org.jboss.windup.operator.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.fabric8.kubernetes.api.model.KubernetesResource;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import lombok.Getter;
+import lombok.Setter;
+
+@JsonDeserialize
+@RegisterForReflection
+@Getter
+@Setter
+public class WindupResourceSpec implements KubernetesResource {
+    private static final long serialVersionUID = 1L;
+
+    private String version;
+    private String application_name;
+    private String hostname_http;
+    private String volumeCapacity;
+    private String mta_Volume_Capacity;
+    private String docker_images_repository;
+    private String docker_images_user;
+    private String docker_images_tag;
+    private String docker_image_web;
+    private String docker_image_executor;
+    private String messaging_serializer;
+    private String db_jndi;
+    private String db_username;
+    private String db_password;
+    private String db_min_pool_size;
+    private String db_max_pool_size;
+    private String db_tx_isolation;
+    private String mq_cluster_password;
+    private String mq_queues;
+    private String mq_topics;
+    private String jgroups_encrypt_secret;
+    private String jgroups_encrypt_keystore;
+    private String jgroups_encrypt_name;
+    private String jgroups_encrypt_password;
+    private String jgroups_cluster_password;
+    private String auto_deploy_exploded;
+    private String sso_url;
+    private String sso_service_url;
+    private String sso_realm;
+    private String sso_username;
+    private String sso_password;
+    private String sso_public_key;
+    private String sso_bearer_only;
+    private String sso_saml_keystore_secret;
+    private String sso_saml_keystore;
+    private String sso_saml_certificate_name;
+    private String sso_saml_keystore_password;
+    private String sso_secret;
+    private String sso_enable_cors;
+    private String sso_saml_logout_page;
+    private String sso_disable_ssl_certificate_validation;
+    private String sso_truststore;
+    private String sso_truststore_password;
+    private String sso_truststore_secret;
+    private String gc_max_metaspace_size;
+    private String max_post_size;
+    private String db_database;
+    private String postgresql_max_connections;
+    private String postgresql_shared_buffers;
+    private String postgresql_cpu_request;
+    private String postgresql_mem_request;
+    private String postgresql_cpu_limit;
+    private String postgresql_mem_limit;
+    private String postgresql_image;
```

Comment:
I can see that we have both `snake_case` and `camelCase`.
Does using one or the other adds information about the fields it's applied to?

Reply:
In fact, snake was chosen in order to be a representation of the env vars used in the template.
So I would fix the camelCase ones to be snake , for the sake of consistency.

Reply:
I suppose it's an habit I have: the usage of `_` reminds me about the constants' naming "convention" (e.g. `CPU_LIMIT`) and I associate camel case to variables.
Right now, my understanding is:

- `snake_case` for environment variables
- `camelCase` for properties set in the deployment

It seems a kind of information we might find useful so let's try to keep it as it is if all my understandings are right.

Reply:
:+1: